backend/app_sdk/app_base.py

```diff
@@ -1214,27 +1214,13 @@ def run_validation(sourcevalue, check, destinationvalue):
                     
                 return False 
             elif check.lower() == "larger than":
-                try:
-                    if sourcevalue.isdigit() and destinationvalue.isdigit():
-                        if int(sourcevalue) > int(destinationvalue):
-                            return True
-
-                    print("Not larger than.")
-                    return False 
-                except AttributeError as e:
-                    self.logger.error("Condition larger than failed with values %s and %s: %s" % (sourcevalue, destinationvalue, e))
-                    return False
+                if str(sourcevalue).isdigit() and str(destinationvalue).isdigit():
```

Comment:
Hey! This is all good, but we need to keep the try/catch as otherwise the program will fail when served values that aren't castable to string. I'll fix this on my end then merge :)